packages/shared/src/components/sidebar/MyFeedAlert.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import XIcon from '@dailydotdev/shared/icons/x.svg';
+import { Alerts } from '../../graphql/alerts';
+import { Button } from '../buttons/Button';
+import { AlertContainer, AlertCopy } from './common';
+import { Pointer, PointerColor } from '../Pointer';
+
+const alertCopy = {
+  migrated: `Psst, your feed has a new name! We've already applied your content filters to it.`,
+  created: `🎉 Your feed is ready! Click here to manage your feed's settings`,
+};
+
+interface MyFeedAlertProps {
+  alerts: Alerts;
+  hideAlert: () => unknown;
+}
+export default function MyFeedAlert({
+  alerts,
+  hideAlert,
+}: MyFeedAlertProps): ReactElement {
+  if (alerts?.filter || (!alerts?.filter && alerts?.myFeed === null)) {
```

Comment:
I'm wondering if this condition should be part of this component or if the parent component should not render it at all if that's the case